conversionFunctions.f90

```diff
@@ -31,42 +31,26 @@ pure function calcM( press, temp ) result ( m )
     real(kind=DP), intent(in) :: press, temp
     real(kind=DP) :: m, press_pa
 
-    press_pa = press * 1.0d+2
-    m = ( 6.02214129d+23 / 8.3144621 ) * ( press_pa / temp )
+    press_pa = press * 1.0d+02
+    m = 1.0d-06 * ( 6.02214129d+23 / 8.3144621 ) * ( press_pa / temp )
```

Comment:
You previously said that this calculation was correct: https://github.com/AtChem/AtChem/issues/133#issuecomment-299840245 . Has this now changed, or is there confusion over the multiplications by 1e6 and 1e-6 at lines 51-2?